pb_plugins/dcsdkgen/type_info.py

```diff
@@ -71,6 +71,20 @@ def inner_name(self):
         else:
             return self._default_types[type_id]
 
+    @property
+    def parent_type(self):
+        """ Extracts parent type. This assumes that nesting is not deeper
+        than 1 level """
+        if self.is_primitive:
+            return None
+
+        type_tree = self._field.type_name.split(".")
+
+        if len(type_tree) <= 5:
```

Comment:
Why 5?

Reply:
Because "org.dronecode.sdk" becomes ".org.dronecode.sdk", and splitting that on `.`'s gives an empty string in the beginning.

I don't really want to hardcode that, but I am not sure how this works and for now it's really based on my observations (they don't really have extensive documentation regarding writing custom protoc plugins). I will improve it as soon as I have more data to understand better where the type comes from :sweat_smile:.

Reply:
Oh, I mean if you can put something like that in the comment that might help the reader.

Reply:
Like that? I added information to the function comment.